extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [UNRELEASED]
 
+- Enable multi-repository variant analysis.
```

Comment:
```suggestion
- Enable multi-repository variant analysis. [#2121](https://github.com/github/vscode-codeql/pull/2121)
```

I believe including a PR link is recommended.

Reply:
Ah thanks!